proto/rpc.proto

```diff
@@ -10,6 +10,7 @@ service Api {
     rpc GetAccountDetails(requests.GetAccountDetailsRequest) returns (responses.GetAccountDetailsResponse) {}
     rpc GetBlockByNumber(requests.GetBlockByNumberRequest) returns (responses.GetBlockByNumberResponse) {}
     rpc GetBlockHeaderByNumber(requests.GetBlockHeaderByNumberRequest) returns (responses.GetBlockHeaderByNumberResponse) {}
+    rpc GetNoteInclusionProofs(requests.GetNotesByIdRequest) returns (responses.GetNoteInclusionProofsResponse) {}
```

Comment:
Even if the requests are the same - we should probably create a different request message (i.e., we probably shouldn't reuse `GetNotesByIdRequest` request here).